src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```
@@ -9,7 +9,8 @@
     <Description>
       OBSOLETE. This package is deprecated and exists only for backwards compatibility.
       Users who need manual instrumentation should reference the "Datadog.Trace" package.
-      ASP.NET is now automatically instrumented.
+      ASP.NET is now automatically instrumented. If your project references this package 
+      or the Datadog.Trace.ClrProfiler.Managed assembly, you can remove them.
```

This is `Datadog.Trace.AspNet`. I was going for "this one or the other one".
```suggestion
      (Datadog.Trace.ClrProfiler.Managed) or Datadog.Trace.AspNet, you can remove them both.
```

---

I see. Ok, I read your intent differently. I'll change it.